qa-core/src/tests/public-api/tables/tables.spec.ts

```diff
@@ -0,0 +1,48 @@
+import TestConfiguration from "../TestConfiguration"
+import PublicAPIClient from "../PublicAPIClient"
+
+describe("Public API - /tables endpoints", () => {
+  let api: PublicAPIClient
+  const config = new TestConfiguration()
+
+  beforeAll(async () => {
+    await config.beforeAll()
+    const app = await config.seedApp()
+    api = new PublicAPIClient(app.data._id)
+  })
+
+  afterAll(async () => {
+    await config.afterAll()
+  })
+
+  it("POST - Create a table", async () => {
+    const response = await api.post(`/tables`, {
+      body: require("./fixtures/table.json")
+    })
+    const json = await response.json()
+    config.testContext.table = json.data
+    expect(response).toHaveStatusCode(200)
+  })
+
+  it("POST - Search tables", async () => {
+    const response = await api.post(`/tables/search`, {
+      body: {
+        name: config.testContext.table.name
```

Comment:
Will this work if this test is run independently? It seems reliant on `config.testContext.table`
If so, should that be done as a part of the `beforeAll` instead?

Reply:
All tests have been updated to seed in the `beforeAll` now 👍 So can be run in isolation